style.css

```diff
@@ -1,4 +1,74 @@
 body {
-background-color: #232323
+  color: #ffffff;
+  font-family: 'Ubuntu', sans-serif;
+  font-size: 15px;
+  width: auto;
+  background: url(nieckovsky_image_main.jpg);
```

Comment:
Aren't we fine just with color here? main image looks like just a single color to me

Reply:
Not necessarily as I want to have a photograph in the background in the end, but since I do not want to use my client's product image in an unfinished project I just used a dummy jpg. I'm not sure if it will behave the same when swapped for just bg color, but I see where this is coming from - for the sake of efficiency I'll make the change for the time being and see if it acts as I need it to.

Reply:
YAGNI Kate :)

Reply:
Really, cutting corners and compromising wherever possible to ensure maximum efficiency resonates deeply with my fluffy heart, it truly does, but honestly, this leaves me puzzled. :)
Since this is supposed to be a photographer's portfolio, it would be nice to have a picture or two outside the gallery itself, just to give clients the first feel of the quality ahead. I've noticed it's a thing lately with them photographers, they like things very visual. ;) Moreover, this is also supposed to be an exercise for me, so I'm trying my best to turn my mockup into a working website and I shouldn't just back out like that, right?

Reply:
Okay you've defended that quite good, I'm game :)